setup.cfg

```diff
@@ -25,10 +25,10 @@ install_requires =
     line-profiler
     numpy>=2.0
     psutil
-    pyuvdata>=3.1.0
+    pyuvdata>=3.1.1
```

Comment:
This probably should be `3.1.2` if you need the new stuff that was added in the latest pyuvdata release.

Reply:
Yep -- thanks for releasing!